src/Skinner.js

```diff
@@ -42,7 +48,25 @@ class Skinner {
                 "If you want to change the active skin, call resetSkin first"
             );
         }
-        this.components = skinObject;
+        this.components = {};
+        var compKeys = Object.keys(skinObject.components);
+        for (var i = 0; i < compKeys.length; ++i) {
+            var comp = skinObject.components[compKeys[i]];
+            this.addComponent(compKeys[i], comp);
+        }
+    }
+
+    addComponent(name, comp) {
+        var slot = name;
+        if (comp.replaces !== undefined) {
```

Comment:
> I like the information that RoomDNDView is a replacement for RoomTile being in RoomDNDView.js itself

I don't like this because I don't want to force `RoomDNDView` to always be clobbering the `RoomTile`. It's impossible to have some DND rooms and some non-DND rooms by adding the mapping to the DND implementation.